src/IssueViz.Security/Store/HotspotsStore.cs

```diff
@@ -81,9 +81,13 @@ private void NotifyHotspotChanged(IAnalysisIssueVisualization changedHotspotViz)
             var hotspotFilePaths = changedHotspotViz
                 .GetAllLocations()
                 .Select(x => x.CurrentFilePath)
+                .Where(x=> !string.IsNullOrEmpty(x))
```

Comment:
Not directly related to the exception but it's still better not to raise the event for the taggers when there's no file path.

Reply:
So this is a bug fix and an separate change of behaviour?

Reply:
It's all under the "don't involve the taggers with null paths" umbrella